marlowe-playground-client/default.nix

```diff
@@ -0,0 +1,49 @@
+{ pkgs, set-git-rev, haskell, docs, easyPS, webCommon, nodejs-headers }:
+
+let
+  playground-exe = set-git-rev haskell.packages.marlowe-playground-server.components.exes.marlowe-playground-server;
+
+  server-invoker =
+    let
+      # the playground uses ghc at runtime so it needs one packaged up with the dependencies it needs in one place
+      runtimeGhc = haskell.packages.ghcWithPackages (ps: [
+        ps.marlowe
+        ps.marlowe-playground-server
+      ]);
+    in
+    pkgs.runCommand "marlowe-server-invoker" { buildInputs = [ pkgs.makeWrapper ]; } ''
+      # We need to provide the ghc interpreter with the location of the ghc lib dir and the package db
+      mkdir -p $out/bin
+      ln -s ${playground-exe}/bin/marlowe-playground-server $out/bin/marlowe-playground
+      wrapProgram $out/bin/marlowe-playground \
+        --set GHC_LIB_DIR "${runtimeGhc}/lib/ghc-${runtimeGhc.version}" \
+        --set GHC_BIN_DIR "${runtimeGhc}/bin" \
+        --set GHC_PACKAGE_PATH "${runtimeGhc}/lib/ghc-${runtimeGhc.version}/package.conf.d" \
+        --set GHC_RTS "-M2G"
+    '';
+
+  generated-purescript = pkgs.runCommand "marlowe-playground-purescript" { } ''
+    mkdir $out
+    ${playground-exe}/bin/marlowe-playground-server psgenerator $out
+  '';
+in
+{
+  inherit server-invoker;
+
+  tutorial = docs.marlowe-tutorial;
```

Comment:
Yes i definitely much prefer this myself. I changed it to only return the client now. I added the `sever-invoker` as a passthru since the docker build needs to access that.